src/pages/Store.vue

```diff
@@ -22,17 +22,15 @@
           tw-whitespace-nowrap
         "
       >
-        {{ $t('store.dappsStore') }}
+        {{ $t('dappStaking.dappStaking') }}
```

Comment:
@impelcrypto Can you also change the page name accordingly?

Reply:
ok,  I've changed.